_layouts/default.html

```diff
@@ -20,17 +20,17 @@
   {% include head.html %}
 
   <body>
-    {% include sidebar.html lang=lang %}
+    {% include_cached sidebar.html lang=lang %}
```

Comment:
I think you should revert this one. The `sidebar.html` template depends on the page information to decide whether to highlight with `active` css class.

So the sidebar will still work without it but this highlight would disappear:
![Screenshot 2023-06-19 at 11 51 29](https://github.com/cotes2020/jekyll-theme-chirpy/assets/322861/6ece410c-162b-439d-9b05-e228bef83abd)

Luckily this template is not to expensive in a large site anyhow:
eg:
```
| Filename                                                           | Count |      Bytes |  Time |
+--------------------------------------------------------------------+-------+------------+-------+
| jekyll-theme-chirpy-6.0.1/_includes/sidebar.html                   |  1168 |   2961.32K | 0.299 |
```

Reply:
Good catch!